system/puppet.py

```diff
@@ -116,6 +126,7 @@ def main():
             facts=dict(default=None),
             facter_basename=dict(default='ansible'),
             environment=dict(required=False, default=None),
+            execute=dict(required=False, default=None),
         ),
         supports_check_mode=True,
         mutually_exclusive=[
```

Comment:
If you list execute in the mutually_exclusive list, ansible can properly complain if someone tries to specify both execute and puppetmaster.